app/views/state_file/questions/nj_medical_expenses/edit.html.erb

```diff
@@ -0,0 +1,46 @@
+<%
+  title = t(".title", filing_year: @filing_year)
+  content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= t(".title") %></h1>
+  <ul class="list--bulleted">
+    <li><%= t(".includes_whom") %></li>
+    <li><%= t(".percent_income") %></li>
+    <li><%= t(".no_reimbursed") %></li>
+  </ul>
+
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <div class="white-group">
+      <%= f.vita_min_money_field(
+        :medical_expenses,
+        t('.label', filing_year: @filing_year),
+        help_text: t('.help_text'),
+        classes: ["form-width--long"],
+        options: { placeholder: "" }
+        # TODO: WHY DOES HELP TEXT OVERLAP LABEL?
```

Comment:
[pebble] If we're okay with merging and fixing these todos in the future (which I'm assuming from the fact that this is in review?) I'd like for each of these TODO comments to be tracked in backlog tickets and not in comments in the code. It's unlikely we're going to be coming back to this file and seeing these in the future

Reply:
+1, one thing I've done in the past is having the backlog ticket in the comment itself

`# TODO github.com/issue/whatever`

Reply:
I would very much prefer to not have the comment exist at all. Not sure having the comment reference a GH issue will improve the usefulness here